src/style-library/demo/StyleDemo.astro

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I don't see the necessary to create an apoc in this part of the code, this code generates too much noise in the pr and the code